app/src/main/res/values/colors.xml

```diff
@@ -11,7 +11,8 @@
     <color name="light_scribe_blue">#FF75CEFA</color>
     <color name="dark_scribe_blue">#FF4CADE6</color>
     <color name="white">#ffffff</color>
-
+    <color name="nav_bar_selected_color_light">@color/dark_scribe_blue</color>
```

Comment:
Just change this to dark_scribe_color which is the background in the dark mode as the color of the navigation bar icons in light mode. Left everything is perfect. Hope we can see you make more amazing contributions to Scribe-Android :) .

Reply:
Got it! Would you also recommend using light_scribe_color for dark mode as well to maintain consistency?

Reply:
Yes , You could do that as well  :)